pagopa/io-app
# feat: [IA-512] Select category screen

## THIS PR DEPENDS ON #3561 
## Short description
This PR add the screen that allow a user to choose a category and if is available a sub-category.

## List of changes proposed in this pull request
- Completed the `ZendeskChooseCategory` screen
- Added the `ZendeskChooseSubCategory` screen


https://user-images.githubusercontent.com/11773070/145539468-9791a53a-904a-4e09-a2e9-24409cf7e350.mp4



## How to test
Follow the flow needed to open a ticket.
Make sure that the `assistanceTool` is zendesk and that the local `zendeskEnabled` FF is true.
